incubator/virtualcluster/go.mod

```diff
@@ -24,8 +26,9 @@ require (
 	k8s.io/cri-api v0.0.0
 	k8s.io/klog v0.3.1
 	k8s.io/kubernetes v1.15.3
-	k8s.io/utils v0.0.0-20190829053155-3a4a5477acf8 // indirect
+	k8s.io/utils v0.0.0-20190829053155-3a4a5477acf8
 	sigs.k8s.io/controller-runtime v0.2.0
+	sigs.k8s.io/controller-tools v0.1.1 // indirect
```

Comment:
please do `go mod tidy` before submit the PR.  `sigs.k8s.io/controller-tools` is not directly imported by code. It would be removed after `go mod tidy`.
Actually, I think it is unnecessary to involve this dependency to go mod. controller-gen just a tool